src/doc/tarpl/vec-raw.md

```diff
@@ -0,0 +1,136 @@
+% RawVec
+
+We've actually reached an interesting situation here: we've duplicated the logic
+for specifying a buffer and freeing its memory. Now that we've implemented it
```

Comment:
I think it'd be good to be more specific about where the duplication is, since this is now it's own page.